youtube_dl/extractor/vidlii.py

```diff
@@ -0,0 +1,129 @@
+                           'contest and no ne of my videos show up so maybe it is broken for everyone else but this one was '
+                           'trying to submit it because I wanted to submit to the contest :) Tanks I hope the website is '
+                           'fixed PS: Jan you are cool please add my video',
+            'thumbnail': 'https://www.vidlii.com/usfi/thmp/tJluaH4BJ3v.jpg',
+            'uploader': 'APPle5auc31995',
+            'url': 'https://cdn.vidlii.com/videos/tJluaH4BJ3v.mp4',
+            'uploader_url': 'https://www.vidlii.com/user/APPle5auc31995',
+            'upload_date': '20171107',
+            'categories': 'News & Politics',
+            'tags': ['Vidlii', 'Jan', 'Videogames'],
+            'duration': 212,
+            'view_count': int,
+            'comment_count': int,
+            'average_rating': float,
+            'type': 'video',
+            'ext': 'mp4'
+        }
+    }, {
+        'url': 'https://www.vidlii.com/watch?v=vBo2IcrwOkO',
+        'md5': 'b42640a596b4dc986702567d49268963',
+        'info_dict': {
+            'id': 'vBo2IcrwOkO',
+            'title': '(OLD VIDEO) i like youtube!!',
+            'description': 'Original upload date:<br />\nMarch 10th 2011<br />\nCredit goes to people who own content in the video',
+            'thumbnail': 'https://www.vidlii.com/usfi/thmp/vBo2IcrwOkO.jpg',
+            'uploader': 'MyEditedVideoSpartan',
+            'url': 'https://cdn.vidlii.com/videos/vBo2IcrwOkO.mp4',
+            'uploader_url': 'https://www.vidlii.com/user/MyEditedVideoSpartan',
+            'upload_date': '20171011',
+            'categories': 'Film & Animation',
+            'tags': None,
+            'duration': 34,
+            'view_count': int,
+            'comment_count': int,
+            'average_rating': float,
+            'type': 'video',
+            'ext': 'mp4'
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        # extract basic properties of video
+        title = self._html_search_regex(r'<h1>(.+?)</h1>', webpage, 'title', default=None) or self._html_search_regex(
+            r'<title>([^<]+?)</title>', webpage, 'title', default="").replace(
+            " - VidLii", "") or self._html_search_meta('twitter:title', webpage, 'title',
+                                                       default="").replace(" - VidLii", "")
+
+        description = strip_or_none(get_element_by_id('des_text', webpage))
+
+        uploader = self._html_search_regex(r'<div[^>]+class="wt_person"[^>]*>(?:[^<]+)<a href="\/user\/[^>]*?>([^<]*?)<',
+                                           webpage, 'uploader', default=None) or self._html_search_regex(
+            r'<img src="[^>]+?class=["\']avt2\s*["\'][^>]+?alt=["\']([^"\']+?)["\']',
+            webpage, 'uploader', default=None)
```

Comment:
Same.

Reply:
FIXED: Merged both REs into single function call. Added compatbility for use of quotes within value, as suggested by you.